jekyll/_posts/2022-02-04-nim-devroom-fosdem-2022.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Nim Devroom at Fosdem 2022"
+author: pietroppeter, pmunch, dom96
+excerpt: "Tomorrow (Sat, Feb 5th) there is a Nim Devroom at Fosdem 2022"
+---
+
+## Nim Devroom at Fosdem 2022
+
+We are happy to announce that all preparations for Nim Devroom at Fosdem 2022 are now complete and
+we are looking forward to 8 excellent talks with more than 5 hours of Nim related content. The conference is online this year
+and the talks  will stream live tomorrow on the Fosdem website. This is the final schedule for the Nim Devroom which will be
+held tomorrow (Saturday, February 5th) from 12:30 to 17:40: https://fosdem.org/2022/schedule/track/nim_programming_language/
+
+**Note that all times are Brussels' times (CET).**
+
+FOSDEM is a two-day event organised by volunteers to promote the widespread use of free and open source software.
```

Comment:
```suggestion
FOSDEM is a two-day event organised by volunteers to promote the widespread use of free and open-source software.
```

Here, open-source is a compound adjective that describes software.

See e.g. https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source